tests/rke/test_upgrade.py

```diff
@@ -115,37 +154,3 @@ def test_upgrade_3(test_name, cloud_provider, rke_client, kubectl):
     validate_rke_cluster(
         rke_client, kubectl, after_upgrade_nodes, 'afterupgrade')
     delete_nodes(cloud_provider, all_nodes)
-
```

Comment:
Can we skip this test case for now and revisit this workflow later.

Reply:
Why should it be skipped ?

Reply:
Skip code added